autoendpoint/src/error.rs

```diff
@@ -296,8 +299,14 @@ impl ResponseError for ApiError {
     fn error_response(&self) -> HttpResponse {
         let mut builder = HttpResponse::build(self.kind.status());
 
-        if self.status_code() == 410 {
-            builder.insert_header(("Cache-Control", "max-age=86400"));
+        match self.status_code() {
+            StatusCode::GONE => {
+                builder.insert_header((header::CACHE_CONTROL, "max-age=86400"));
```

Comment:
not blocking on this but FYI there's also this fancy way of doing it:

```suggestion
                builder.insert_header(CacheControl(vec![CacheDirective::MaxAge(86400)]));
```
```
use actix_web::{
    http::header::{CacheControl, CacheDirective}
};
```